Dockerfile

```diff
@@ -81,6 +81,7 @@ FROM base AS lint-golangci-lint
 WORKDIR /src
 COPY .golangci.yml .
 ENV GOGC 50
+RUN golangci-lint config verify --config .golangci.yml
```

Comment:
hmm I guess we can optimize this part, have a different stage that just does verify, so we don't re-verify for any source code change and only for changes to golangci.yml

Reply:
i think we could optimize the dockerfile anyways. let's track it separately